packaging/test_irods_client_rest_cpp.py

```diff
@@ -190,6 +191,98 @@ def test_put_configuration(self):
         os.remove(file1)
         os.remove(file2)
 
+    def test_meta_data_object(self):
```

Comment:
Use more words to describe what the test is doing. 

Consider changing to `test_add_and_remove_metadata_operations_on_data_object__issue_NNNN`

Where `NNNN` is the issue number.

Same thing for other tests.